### DISCUSSION TITLE
volk.dll soapy.dll missing

### DISCUSSION THREAD

**Discussion by Sperhak-Industries** [AlexandreRouma/SDRPlusPlus#139] [Q&A] [answered]
Hi i canot run program because volk.dll soapy.dll missing how to intall it?

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
From where did you download the program? The author recommended in #128 that you should download it from the latest CI run. Right now the latest successful CI run is here: https://github.com/AlexandreRouma/SDRPlusPlus/actions/runs/812047935. Open the link and scroll down to Artifacts, then download `sdrpp_windows_x64` (assuming you are on Windows because you mentioned dll files).

**Reply by Sperhak-Industries** [reply]
Many thanks this working.I download SDRPlusPlus_Release_v0.2.4_alpha.zip

**Reply by the-gobleo-tinkerer** [reply]
That's good to hear. Indeed 0.2.4 is quite old and may have issue running. If there are no further problems would you mark this question as "answered" so that others can see the problem has been solved?
